docs/Upgrades and Downgrades.md

```diff
@@ -89,3 +89,4 @@ more.
 
 To generate an upgrade release, you'll need to pass `--upgrade` to `mix release`. To generate an upgrade from an arbitrary
 version you've previously built, pass `--upfrom=<version>`. Distillery will look for the appup in the ebin of your current build.
+If you want soft purge during the upgrade/downgrade, you could use `--soft-purge`.
```

Comment:
Can you expand the docs to here to explain the difference between soft purge and brutal purge, and why you would choose soft purges? It will certainly be the first thing people will want to know when they see the option :).

I would change the text here to read "If you want to use soft purges rather than the default brutal purge of old module versions during upgrades/downgrades, you can pass `--soft-purge` or set `soft_purge: true` in the release config file."